Terminal.Gui/Views/Menu.cs

```diff
@@ -2099,5 +2099,30 @@ public override bool OnEnter (View view)
 
 			return base.OnEnter (view);
 		}
+
+		/// <summary>
+		/// Gets the superview location offset relative to the <see cref="ConsoleDriver"/> size 
+		/// which is measured by the <see cref="ConsoleDriver.Cols"/> and <see cref="ConsoleDriver.Rows"/>.
+		/// </summary>
+		/// <returns>The location offset.</returns>
+		internal Point GetDriverLocationOffset ()
+		{
+			var superViewFrame = SuperView == null ? new Rect (0, 0, Driver.Cols, Driver.Rows) : SuperView.Frame;
+			var sv = SuperView == null ? Application.Current : SuperView;
+			return new Point (superViewFrame.X - sv.Frame.X - (superViewFrame.X != sv.Frame.X ? 1 : 0),
+				superViewFrame.Y - sv.Frame.Y - (superViewFrame.Y != sv.Frame.Y ? 1 : 0));
+		}
+
+		/// <summary>
+		/// Gets the <see cref="Application.Current"/> location offset relative to the <see cref="ConsoleDriver"/> size.
+		/// </summary>
+		/// <returns>The location offset.</returns>
+		internal Point GetDriverLocationOffsetFromCurrent ()
+		{
+			var topFrame = new Rect (0, 0, Driver.Cols, Driver.Rows);
+			var currentFrame = Application.Current.Frame;
+			return new Point (topFrame.X - currentFrame.X - (topFrame.X != currentFrame.X ? 1 : 0),
```

Comment:
What is the significance of the `- 1` or `- 0`?

Reply:
It's the bounds offset. I'll change that.